cedar-drt/fuzz/fuzz_targets/abac-type-directed.rs

```diff
@@ -78,6 +78,20 @@ impl<'a> Arbitrary<'a> for FuzzTargetInput {
         ];
         let all_entities = Entities::try_from(hierarchy).map_err(|_| Error::NotEnoughData)?;
         let entities = drop_some_entities(all_entities, u)?;
+        let validator_schema =
+            cedar_policy_core::validator::ValidatorSchema::try_from(schema.clone())
+                .map_err(|e| Error::SchemaError(e))?;
+        let actions = validator_schema
+            .action_entities()
+            .map_err(|_| Error::EntitiesError("Error fetching action entities".into()))?;
+        let entities = entities
+            .add_entities(
+                actions.into_iter().map(|e| std::sync::Arc::new(e)),
+                None::<&cedar_policy_core::validator::CoreSchema>,
+                cedar_policy_core::entities::TCComputation::AssumeAlreadyComputed,
+                cedar_policy_core::extensions::Extensions::all_available(),
+            )
+            .map_err(|_| Error::EntitiesError("Error adding action entities to entities".into()))?;
```

Comment:
would probably be helpful to include the text of the underlying error, something like
```suggestion
            .map_err(|e| Error::EntitiesError(format!("Error adding action entities to entities: {e}")))?;
```